cf-templates/osi.yaml

```diff
@@ -0,0 +1,419 @@
+      VpcOptions: 
+        SecurityGroupIds:
+          - !Ref OpenSearchIngestionPipelineSecurityGroup
+        SubnetIds:
+          - !Ref PrivateSubnet1
+          - !Ref PrivateSubnet2
+          - !Ref PrivateSubnet3
+
+  OtelLogsPipeline:
+    Type: "AWS::OSIS::Pipeline"
+    DependsOn:
+      - OpenSearchIngestionPipelineRole
+      - OpenSearchIngestionPipelineLogGroup
+      - OpenSearchIngestionPipelineSecurityGroup
+    Properties:
+      LogPublishingOptions:        
+        IsLoggingEnabled: True
+        CloudWatchLogDestination:
+          LogGroup: !Ref OpenSearchIngestionPipelineLogGroup
+      MaxUnits: 4
+      MinUnits: 1
+      PipelineConfigurationBody:
+        !Sub |
+          version: "2"
+          otel-logs-pipeline:
+            source:
+              otel_logs_source:
+                path: "/v1/logs"
+            processor:
+              - parse_json:
+                  source: "body"
+                  parse_when: "/traceId ==\"\""
+              - parse_json:
+                  source: "kubernetes"
+                  parse_when: "/traceId ==\"\""
+              - parse_json:
+                  source: "annotations"
+                  parse_when: "/traceId ==\"\""
+              - parse_json:
+                  source: "labels"
+                  parse_when: "/traceId ==\"\""
+              - delete_entries:
+                  with_keys: ["body", "kubernetes", "annotations","labels"]
+              - date:
+                  from_time_received: true
+                  destination: "@timestamp"           
+            sink:
+              - opensearch:                  
+                  index: "sample_app_logs"
+                  hosts: ["https://${AOSDomainEndpoint}"]
+                  aws:                  
+                    sts_role_arn: "${OpenSearchIngestionPipelineRole.Arn}"
+                    region: ${AWS::Region}                
+      PipelineName: !Sub ${PipelineName}-otellogs
+      VpcOptions: 
+        SecurityGroupIds:
+          - !Ref OpenSearchIngestionPipelineSecurityGroup
+        SubnetIds:
+          - !Ref PrivateSubnet1
+          - !Ref PrivateSubnet2
+          - !Ref PrivateSubnet3
+
+  OtelMetricsPipeline:
+    Type: "AWS::OSIS::Pipeline"
+    DependsOn:
+      - OpenSearchIngestionPipelineRole
+      - OpenSearchIngestionPipelineLogGroup
+      - OpenSearchIngestionPipelineSecurityGroup
+    Properties:
+      LogPublishingOptions:        
+        IsLoggingEnabled: True
+        CloudWatchLogDestination:
+          LogGroup: !Ref OpenSearchIngestionPipelineLogGroup
+      MaxUnits: 4
+      MinUnits: 1
+      PipelineConfigurationBody:
+        !Sub |
+          version: "2"
+          otel-metrics-pipeline:
+            source:
+              otel_metrics_source:
+                path: "/v1/metrics"
+            processor:
+              - otel_metrics:
+            sink:
+              - opensearch:
+                  index: "metrics"
```

Comment:
I suggest changing to:

`index: "sample_app_metrics"`

Reply:
Remembering that we need to create an issue to add scrap the metrics through OpenTelemetry and also add it to the application.

[Ref-01](https://opentelemetry-python.readthedocs.io/en/latest/api/metrics.html)

Reply:
changed index name